src/style/app.scss

```diff
@@ -32,6 +32,10 @@ $govuk-compatibility-govukfrontendtoolkit: true;
 @import "./vendor/prism.scss";
 @import "./material";
 
+// Ensure Media Viewer CSS styles and icons are available globally across Manage Cases. Required for correct display
+// of the Media Viewer toolbar when this component is embedded in another CCD Case UI Toolkit component
+@import '../../node_modules/@hmcts/media-viewer/assets/all';
```

Comment:
Doesn't `@import '@hmcts/media-viewer/assets/all';` OR `@import '~@hmcts/media-viewer/assets/all';` work?

Reply:
You're correct, @MateiHMCTS - these suggestions should have worked but, for some reason, they didn't when I tried them. I might give them another try

Reply:
@MateiHMCTS - tried again and they do work; I'll update the import accordingly